src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -121,7 +121,8 @@ public class RedisHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposab
 
                 if (groupMessage.Action == GroupAction.Remove)
                 {
-                    if (!await RemoveGroupAsyncCore(groupMessage.ConnectionId, groupMessage.Group))
+                    var connection = _connections[groupMessage.ConnectionId];
+                    if (connection == null || !await RemoveGroupAsyncCore(connection, groupMessage.Group))
```

Comment:
Can you put parents around `await RemoveGroupAsyncCore(connection, groupMessage.Group)`. Looks weird to me to see the ! next to the await